README.md

```diff
@@ -2,15 +2,21 @@
 
 # TorchCodec
 
-TorchCodec is a Python package that provides easy-to-use and fast APIs to
-decode video frames to PyTorch Tensors. These tensors can then be transformed
-and fed to ML models. TorchCodec needs a working FFMPEG installation (that the
-user provides) and uses it as a library to do the decoding. We use FFMPEG
-because of the diverse video formats it supports as well as its popularity.
-
-Under the hood, TorchCodec uses Pytorch's C++ custom ops to wrap C++ code that
-does the memory management of packets, frames and codecs. TorchCodec hides
-that complexity under its API (see example usage below).
+TorchCodec is a Python library for decoding videos into PyTorch tensors. It aims 
+to be fast, easy to use and well integrated into the PyTorch ecosystem. If you 
```

Comment:
hyphenated seems better

Reply:
Which phrase?

Reply:
oxford comma after "use", I think

Reply:
Oh,  @ahmadsharif1 is referring to "easy to use." That should only be hyphenated when it modifieds a noun, as it is a phrase that acts as an adjective. But we're using the phrase in its verb form, so it shouldn't be hyphenated.

And I grudgingly accept that I am in the minority of disliking Oxford commas. :)

Reply:
Yes, was talking about "easy-to-use".

Otherwise it could be parsed as "fast, easy" as well, leaving the "to use" a bit confusing.